PurchasesTests/Purchasing/PurchaserInfoTests.swift

```diff
@@ -346,11 +350,17 @@ class BasicPurchaserInfoTests: XCTestCase {
         expect(entitlements["old_pro"]).to(beNil());
     }
 
-    func testPurchaseDate() {
+    func testPurchaseDateForEntitlement() {
         let purchaseDate = self.purchaserInfo!.purchaseDate(forEntitlement: "pro")
```

Comment:
Since you did it in testParsesExpirationDate, I think we should also add the XCTUnwrap for `self.purchaserInfo`.